wmfdata/metadata.py

```diff
@@ -4,4 +4,4 @@
 # installed.
 
 source = "https://github.com/wikimedia/wmfdata-python"
-version = "2.0.1"
+version = "2.0.2"
```

Comment:
Let's remove this change for now, and later we can do a release via https://github.com/wikimedia/wmfdata-python/blob/main/docs/developing.md#releasing-a-new-version

Reply:
Done.